mobile-app/app/screens/AppNavigator/screens/Balances/components/BalanceControlCard.tsx

```diff
@@ -0,0 +1,78 @@
+import { tailwind } from '@tailwind'
+import { ThemedIcon, ThemedText, ThemedView } from '@components/themed'
+import React from 'react'
+import { useWalletContext } from '@shared-contexts/WalletContext'
+import { View } from '@components'
+import { TouchableOpacity } from 'react-native'
+import { translate } from '@translations'
+import { useDeFiScanContext } from '@shared-contexts/DeFiScanContext'
+import { openURL } from '@api/linking'
+// @ts-expect-error
+import Avatar from 'react-native-boring-avatars'
+import { NavigationProp, useNavigation } from '@react-navigation/native'
+import { BalanceParamList } from '@screens/AppNavigator/screens/Balances/BalancesNavigator'
+import { IconButton } from '@components/IconButton'
+
+export function BalanceControlCard (): JSX.Element {
+  const { address } = useWalletContext()
+  const { getAddressUrl } = useDeFiScanContext()
+  const navigation = useNavigation<NavigationProp<BalanceParamList>>()
+  return (
+    <ThemedView
+      testID='balance_control_card'
+      style={tailwind('p-4 flex flex-col')}
+      dark={tailwind('bg-gray-800 border-b border-gray-700')}
+      light={tailwind('bg-white border-b border-gray-100')}
+    >
+      <View style={tailwind('flex flex-row items-center')}>
+        <Avatar
+          size={30}
```

Comment:
![image](https://user-images.githubusercontent.com/40191153/140016832-0d10eec7-9160-4cfa-aabc-5063b2989f8f.png)
Figma uses size 40 tho, probably to distinguish with the token icon size (32) under the balances list

Reply:
Updated